addons/hr_skills/models/hr_skill_level.py

```diff
@@ -39,3 +39,10 @@ def write(self, vals):
         if vals.get('default_level'):
             self.skill_type_id.skill_level_ids.filtered(lambda r: r.id != self.id).default_level = False
         return res
+
+    def copy(self, default=None):
```

Comment:
Could be achievable by overriding copy_data right ? So before creating the records themselves.